src/m.tex

```diff
@@ -99,7 +99,20 @@ \section{Division Operations}
 DIV and DIVU perform an XLEN bits by XLEN bits signed and unsigned integer
 division of {\em rs1} by {\em rs2}, rounding towards zero.
 REM and REMU provide the remainder of the
-corresponding division operation.  If both the quotient and remainder
+corresponding division operation.
+For signed division, the sign of the remainder and the value of the quotient
+follow the usual conventions: the sign of the remainder is the sign of the
+dividend and the absolute value of the quotient is the same as the quotient for
```

Comment:
I believe the remark about the absolute value of the quotient is redundant with the previous sentence ("rounding towards zero").  Agreed?

Reply:
Could move it to the commentary you added below (e.g. "Rounding towards zero implies that the abs val of the quotient is the same as [...].  In combination with the sign of the remainder, this ensures that [...]")

Reply:
I convinced myself that the quotient definition follows from the rounding scheme.  I'm committing a change that includes the sign-of-REM clarification and the commentary below.